v1.1.1/js/Loopy.js

```diff
@@ -30,6 +30,7 @@ function Loopy(config){
 
 	// Mouse
 	Mouse.init(document.getElementById("canvasses")); // TODO: ugly fix, ew
+	Mouse.init(document.getElementById("graph_canvas")); // TODO: This makes the graph know when it's hovered, but also allows you to draw on the graph. What do?
```

Comment:
This line is what enables the graph sidebar to respond to clicks like any other element of the diagram (such as nodes, edges, labels). However, it's also what causes us to be able to draw on the graph canvas :(

The only real way that I can think of to get around this at the moment is to add a bunch of conditionals to `Mouse.js` so that it doesn't add listeners for mousemove or mouseup on the graph_canvas, but it feels clumsy